wled00/xml.cpp

```diff
@@ -328,13 +328,23 @@ void getSettingsJS(byte subPage, char* dest)
     sappend('v',SET_F("AC"),apChannel);
     sappend('c',SET_F("WS"),noWifiSleep);
 
+    #ifndef WLED_DISABLE_REMOTE
+    sappend('c',SET_F("RE"),enable_espnow_remote);
+    sappends('s',SET_F("RMAC"),linked_remote);
+    #endif
+
     #ifdef WLED_USE_ETHERNET
     sappend('v',SET_F("ETH"),ethernetType);
     #else
     //hide ethernet setting if not compiled in
     oappend(SET_F("document.getElementById('ethd').style.display='none';"));
     #endif
 
+    #ifdef WLED_DISABLE_REMOTE
```

Comment:
Why not use #else from lines above (334)?

Reply:
Just oversight on my part, thanks for catching